ts/packages/anchor/src/provider.ts

```diff
@@ -31,7 +33,8 @@ export default interface Provider {
   sendAndConfirm?(
     tx: Transaction | VersionedTransaction,
     signers?: Signer[],
-    opts?: ConfirmOptions
+    opts?: ConfirmOptions,
+    blockhash?: BlockhashWithExpiryBlockHeight
```

Comment:
It would be better to include `blockhash` in `opts` rather than introducing another parameter. Otherwise it would require passing `undefined` if we don't want to change `opts` which is not ideal.